.github/ISSUE_TEMPLATE/tutorials-bug-report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Cosmos SDK tutorials bug report
+about: Create a report to help us improve
+title: ''
+labels: report
+assignees: ''
+
+---
+
+**What tutorial are you working in?**
+Paste the URL of the tutorial.
+
+**Describe the bug**
+Tell us about your challenges. A clear and concise description of your problem helps us reproduce the issue. 
+
```

Comment:
When working with starport, please report your version. You can check with `starport version`.